src/patternfly/components/List/list.scss

```diff
@@ -1,11 +1,29 @@
 .pf-c-list {
+  // list
   --pf-c-list--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-list--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-list--nested--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-list--ul--ListStyle: var(--pf-global--ListStyle);
   --pf-c-list--li--MarginTop: var(--pf-global--spacer--sm);
+
+  // inline
   --pf-c-list--m-inline--li--MarginRight: var(--pf-global--spacer--lg);
 
+  // bordered
+  --pf-c-list--m-bordered--li--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-list--m-bordered--li--BorderBottomColor: var(--pf-global--BorderColor--100);
+  --pf-c-list--m-bordered--li--BorderBottomWidth: var(--pf-global--BorderWidth--sm);
+
+  // icon
+  --pf-c-list__item-icon--MinWidth: var(--pf-global--FontSize--xs);
+  --pf-c-list__item-icon--MarginTop: #{pf-size-prem(6px)};
+  --pf-c-list__item-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-list__item-icon--Color: var(--pf-global--icon--Color--light);
+  --pf-c-list__item-icon--FontSize: var(--pf-global--icon--FontSize--sm);
+  --pf-c-list--m-lg__item-icon--MinWidth: var(--pf-global--FontSize--3xl);
```

Comment:
The same comment here about the `min-width` - it should match what the font-size is so that the space provided works for a square icon or one more narrow than it is tall.